test/unit/template_source_tests.rb

```diff
@@ -127,6 +110,14 @@ class RenderTests < InitTests
       end
     end
 
+    should "raise an argument error if name matches multiple templates" do
```

Comment:
@jcredding maybe say "complain if the given template name matches..." - better matches other "complain" test descriptions.